include/common/aarch32/el3_common_macros.S

```diff
@@ -195,6 +195,18 @@
 	cps	#MODE32_mon
 	isb
 
+#if defined(IMAGE_BL32) && SP_MIN_WITH_SECURE_FIQ
```

Comment:
Following discussion from https://github.com/ARM-software/arm-trusted-firmware/pull/977#pullrequestreview-46814003, i will move back this SCR init so that it remains in the sp_min in **bl32/sp_min/aarch32/entrypoint.S**.